addon/components/o-s-s/avatar.hbs

```diff
@@ -0,0 +1,8 @@
+<div class={{this.computedClass}} ...attributes>
+  {{#if this.displayInitial}}
+    <span class="text-style-uppercase">{{@initials}}</span>
+  {{else}}
+    <img src={{this.image}} alt={{t "oss-components.badge.image_alt"}} class={{this.imageClass}}
+         onerror={{this.imageLoadError}} loading="lazy" />
```

Comment:
`loading="lazy"`
Are you sure this is the behaviour we want here ? Did you check with anyone ?

Reply:
Julien and Philippe said me to put that